src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -982,8 +949,8 @@ void GcsActorManager::LoadInitialData(const EmptyCallback &done) {
         // We should not reschedule actors in state of `ALIVE`.
         // We could not reschedule actors in state of `DEPENDENCIES_UNREADY` because the
         // dependencies of them may not have been resolved yet.
-        RAY_LOG(DEBUG) << "Rescheduling a non-alive actor, actor id = "
-                       << actor->GetActorID() << ", state = " << actor->GetState();
+        RAY_LOG(INFO) << "Rescheduling a non-alive actor, actor id = "
+                      << actor->GetActorID() << ", state = " << actor->GetState();
```

Comment:
Btw, it would probably be best if we had some common utility or macro that we could use to append these to the logs (or maybe something like a logging context that gets initialized in the process). What do you think?

Reply:
That way we could keep the formatting and information in sync everywhere

Reply:
@edoakes Thanks, do you mean add a DebugString function to class GcsActor?